gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/application/use_case/ImportApplicationCRDUseCase.java

```diff
@@ -51,30 +67,41 @@ public class ImportApplicationCRDUseCase {
     private final ImportApplicationCRDDomainService importApplicationCRDDomainService;
     private final ApplicationMetadataCrudService applicationMetadataCrudService;
     private final ApplicationMetadataQueryService applicationMetadataQueryService;
+    private final MemberQueryService memberQueryService;
+    private final UserDomainService userDomainService;
 
     public ImportApplicationCRDUseCase(
         ApplicationCrudService applicationCrudService,
         ImportApplicationCRDDomainService importApplicationCRDDomainService,
         ApplicationMetadataCrudService applicationMetadataCrudService,
-        ApplicationMetadataQueryService applicationMetadataQueryService
+        ApplicationMetadataQueryService applicationMetadataQueryService,
+        MemberQueryService memberQueryService,
+        UserDomainService userDomainService
     ) {
         this.applicationCrudService = applicationCrudService;
         this.importApplicationCRDDomainService = importApplicationCRDDomainService;
         this.applicationMetadataCrudService = applicationMetadataCrudService;
         this.applicationMetadataQueryService = applicationMetadataQueryService;
+        this.memberQueryService = memberQueryService;
+        this.userDomainService = userDomainService;
     }
 
     public record Output(ApplicationCRDStatus status) {}
 
     public record Input(AuditInfo auditInfo, ApplicationCRDSpec crd) {}
 
     public Output execute(Input input) {
+        BaseApplicationEntity application;
         if (input.crd.getId() == null) {
```

Comment:
I don't know about that one. When we will get the export, there will be an ID in the CRD even if the operator "creates" it, and an update should be perform.

Maybe we should use the same pattern as we use for API WDYT ? (setting the ID or generating it in the operator, and look there if there is something to update or not as done below)